xml/constraints-next.rng

```diff
@@ -88,7 +94,12 @@
             </attribute>
           </optional>
           <optional>
-            <externalRef href="score.rng"/>
+            <choice>
+              <externalRef href="score.rng"/>
+              <attribute name="kind">
+                <ref name="order-types"/>
+              </attribute>
+             </choice>
```

Comment:
FYI No harm in updating this, but I plan to drop the -next schema at the next release series